src/test/regress/expected/materialized_view.out

```diff
@@ -200,3 +200,74 @@ SELECT count(*) FROM materialized_view;
 DROP MATERIALIZED VIEW materialized_view;
 DROP SCHEMA materialized_view CASCADE;
 NOTICE:  drop cascades to view air_shipped_lineitems
+-- modify statements on/with views
+-- create two tables and a view
+CREATE TABLE large (id int, tenant_id int);
+CREATE TABLE small (id int, tenant_id int);
+SELECT create_distributed_table('large','tenant_id');
+ create_distributed_table 
+--------------------------
+ 
+(1 row)
+
+SELECT create_distributed_table('small','tenant_id');
+ create_distributed_table 
+--------------------------
+ 
+(1 row)
+
+CREATE MATERIALIZED VIEW small_view AS SELECT * from small where id < 100;
+\copy small FROM STDIN DELIMITER ','
+\copy large FROM STDIN DELIMITER ','
+-- running modify statements "on" views is still not supported, hence below two statements will fail
+UPDATE small_view SET id = 1;
+ERROR:  cannot change materialized view "small_view"
+DELETE FROM small_view;
+ERROR:  cannot change materialized view "small_view"
+-- using views in modify statements' FROM / WHERE clauses is stll valid 
+UPDATE large SET id=20 FROM small_view WHERE small_view.id=large.id; --rows with id=6 and id=1 now become id=20 in large table
```

Comment:
what happens if the view contains join of two tables and users updates the view? First, can you check what does postgres do?

Reply:
```sql
ERROR:  cannot update view "localtablejoin"
DETAIL:  Views that do not select from a single table or view are not automatically updatable.
HINT:  To enable updating the view, provide an INSTEAD OF UPDATE trigger or an unconditional ON UPDATE DO INSTEAD rule.
```

As I see, we even cannot update views that contain join of two local tables in postgresql automatically.